### PR TITLE
Add ci scripts

### PR DESCRIPTION
**Pull request by vparfonov**
Signed-off-by: Vitalii Parfonov <vparfono@redhat.com>

### What does this PR do?

Preparation for moving CI jobs from Codenvy Ci to the CentOS CI

### CODE REVIEW

**Comment by themr0c** [comment]
Hi, this is not a maven project, I doubt this is supposed to do something useful.

What is the CentOS CI based on?

**Comment by skabashnyuk** [comment]
Docs is a part of the Che assembly https://github.com/eclipse/che/blob/master/assembly/assembly-main/pom.xml#L55-L60.

File: .ci/cico_common.sh
Comment (ibuziuk) [inline .ci/cico_common.sh:105]: no new line in the end

File: .ci/cico_common.sh
Comment (ibuziuk) [inline .ci/cico_common.sh:64]: @vparfonov can we push an image to https://quay.io/organization/eclipse ?

**Comment by ibuziuk** [comment]
> Hi, this is not a maven project, I doubt this is supposed to do something useful.

@themr0c why there is a pom.xml inside the repo in this case ?

**Comment by themr0c** [comment]
We are building on Eclipse CI, we do not need to move to Centos CI.

Most probably we will decide to move to new Eclipse CI in the future.

**Comment by themr0c** [comment]
Seems there is a job on codeenvy CI afterall. But it is not the one used to publish the docs on the website. I was not even aware of that, sorry.

**Comment by themr0c** [comment]
Is the build on codeenvy CI building the documentation to embed it in the che distribution?

**Comment by skabashnyuk** [comment]
>Is the build on codeenvy CI building the documentation to embed it in the che distribution?

I think yes. @vparfonov right?

**Comment by vparfonov** [comment]
@themr0c yes
At the moment we have this CI job for che-docs repository https://ci.codenvycorp.com/job/che-docs-ci-master/ it runs on each commit (`mvn clean install`).
Here produced artifacts: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/che/docs/che-docs/.
Also, as @skabashnyuk mention it used as maven dependency. 
So at the moment its Maven project for us, if you want to change that it can't be solved just in CI builds scripts

**Comment by themr0c** [comment]
Okay, something we have to pay attention to when moving to antora. It means we have currently 2 distinct build processes: one to produce a .war that is reused by che, one to produce the docs website.

**Comment by vparfonov** [comment]
@themr0c So I can close this PR, as not actual?

**Comment by themr0c** [comment]
Unfortunately, the build producing the war that is embedded into the application needs to be migrated. So this PR is actual. But I am not the right person to test it and approve.

**Review by themr0c — APPROVED** [review]
I will wait for another approval to merge. I have no way to test it completely.

**Pull request merged**
